packages/react-components/react-teaching-popover-preview/src/components/TeachingPopover/renderTeachingPopover.tsx

```diff
@@ -1,58 +1,16 @@
 import * as React from 'react';
-import { PopoverProvider } from '@fluentui/react-popover';
-import { TeachingPopoverProvider } from '../../TeachingPopoverContext';
-import type { TeachingPopoverContextValues, TeachingPopoverState } from './TeachingPopover.types';
+import { PopoverContextValue, PopoverProvider } from '@fluentui/react-popover';
+import type { TeachingPopoverState } from './TeachingPopover.types';
 
 /**
  * Render the final JSX of TeachingPopover
  */
-export const renderTeachingPopover_unstable = (
-  state: TeachingPopoverState,
-  contextValues: TeachingPopoverContextValues,
-) => {
-  const {
-    appearance,
-    arrowRef,
-    contentRef,
-    inline,
-    mountNode,
-    open,
-    openOnContext,
-    openOnHover,
-    setOpen,
-    size,
-    toggleOpen,
-    trapFocus,
-    triggerRef,
-    withArrow,
-    inertTrapFocus,
-  } = state;
-
+export const renderTeachingPopover_unstable = (state: TeachingPopoverState, contextValues: PopoverContextValue) => {
```

Comment:
Out of curiosity, can't we do `export const renderTeachingPopover_unstable = renderPopover_unstable`?

Reply:
This was mostly an attempt to move to the above format where context is passed in from the wrapper class, I'll set this to be equal to renderPopover for now, but would probably be good to update Popover and this as a seperate PR